nuget/NUnit3TestAdapter.nuspec

```diff
@@ -24,6 +24,19 @@
     <tags>test visualstudio testadapter nunit nunit3 dotnet</tags>
 
     <developmentDependency>false</developmentDependency>
+
+    <dependencies>
+      <group targetFramework="net462">
+        <dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="1.0.1" />
+        <dependency id="Microsoft.Testing.Extensions.Telemetry" version="1.0.1" />
+        <dependency id="Microsoft.Testing.Platform.MSBuild" version="1.0.1" />
```

Comment:
This package contains the main logic for handling MSBuild: producing the entry point, registering and loading extensions... This is not required but without it, the user would have to manually provide the entry point.

This is also handling the `dotnet test` logic and many more MSBuild related features.